codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/ServiceGenerator.java

```diff
@@ -398,14 +398,13 @@ defer endTimer()
                     decorated := middleware.DecorateHandler(handler, stack)
                     result, metadata, err = decorated.Handle(ctx, params)
                     if err != nil {
-                        span.SetProperty("error.go.type", fmt.Sprintf("%T", err))
-                        span.SetProperty("error.go.error", err.Error())
+                        span.SetProperty("exception.type", fmt.Sprintf("%T", err))
+                        span.SetProperty("exception.message", err.Error())
 
                         var aerr smithy.APIError
                         if $errors.As:T(err, &aerr) {
-                            span.SetProperty("error.api.code", aerr.ErrorCode())
-                            span.SetProperty("error.api.message", aerr.ErrorMessage())
-                            span.SetProperty("error.api.fault", aerr.ErrorFault().String())
```

Comment:
What's fault in the context of Smithy?

Reply:
"client", "server" or "unknown"

Reply:
oh, like "who's fault is this". 

I may be biased towards the errors I happen to be looking at right now, but this does seem useful for communicating with customers and saying "this is a client error, not an issue with the server", although I'd defer to your experience if you think this is not important

Reply:
Honestly, I'm fine leaving it